pages/common/pprof.md

```diff
@@ -0,0 +1,24 @@
+# pprof
+
+> Comand-line tool for visualization and analysis of profiling data.
+> More information: <https://github.com/google/pprof>.
+
+- Generate a text report from fibbo-profile.pb.gz profile data on fibbo binary:
```

Comment:
fibbo-profile.pb.gz within backticks here and after. And no need for "profile data". It is understood with the name that it is a profile.